tests/test_views.py

```diff
@@ -101,6 +102,15 @@ def test_get_succeeds(self):
         assert response.status_code == status.HTTP_200_OK
         assert response.data == {'method': 'GET'}
 
+    def test_logged_in_get_succeeds(self):
+        user = User.objects.create_user('user', 'user@example.com', 'password')
+        request = factory.get('/')
+        del user.is_active
+        request.user = user
```

Comment:
Question: I thought the error comes from the `sessionid` authentication part. Here, you are directly giving the request the authed user, which is already past the error part?

Reply:
The error is due to the ORM making a query, not just because of the session authentication. This is the easiest way I could reproduce the error.